devito/dle/backends/basic.py

```diff
@@ -107,7 +107,7 @@ def _create_elemental_functions(self, nodes, state):
             args.extend([(i.name, Scalar(name=i.name, dtype=i.dtype)) for i in required])
 
             call, params = zip(*args)
-            handle = flatten([p.rtargs for p in params])
+            handle = flatten(params)
```

Comment:
the ``handle`` variable can at this point be dropped and its rvalue inlined at 117

Reply:
Fixed

Reply:
Look again